common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4107,6 +4107,12 @@
     ],
     "sqlState" : "38000"
   },
+  "RANDSTR_LENGTH_NOT_POSITIVE_OR_ZERO" : {
```

Comment:
Could you add a sub-class of `INVALID_PARAMETER_VALUE`, please.

Reply:
Thanks, good suggestion. I looked and found that `INVALID_PARAMETER_VALUE.LENGTH` already exists, so we can just reuse that.